README.md

```diff
@@ -98,4 +104,17 @@ go get -u github.com/sourcegraph/src-cli/cmd/src
 3.  Travis will automatically perform the release. Once it has finished, **confirm that the curl commands fetch the latest version above**.
 4.  Update the `MinimumVersion` constant in the [src-cli package](https://github.com/sourcegraph/sourcegraph/tree/master/internal/src-cli/consts.go).
 
-**Note**: The version recommended by a Sourcegraph instance will be the highest patch version with the same major and minor version as the set minimum. This means that patch versions are reserved solely for non-breaking changes and minor bug fixes. This allows us to dynamically release fixes for older versions of src-cli without having to update the instance.
+### Patch releases
+
+The version recommended by a Sourcegraph instance will be the highest patch version with the same major and minor version as the set minimum. This means that patch versions are reserved solely for non-breaking changes and minor bug fixes. This allows us to dynamically release fixes for older versions of src-cli without having to update the instance.
```

Comment:
I think we can simplify this a bit by avoiding passive voice:

```suggestion
A Sourcegraph instance returns the highest patch version with the same major and minor version as the set minimum. Patch versions are reserved solely for non-breaking changes and minor bug fixes. This allows us to dynamically release fixes for older versions of `src-cli` without having to update the instance.
```

I also don't see any definition of "set minimum" anywhere in scope. What I think we are saying is "the same major and minor version as the oldest compatible version" (similar to minimum version selection in Go). Is that correct?

Reply:
Updated.